docs(Configuration): fix document amdContainer (#6749)
webpack/webpack.js.org: src/content/configuration/output.mdx

<Badge text="5.78.0+" />

> Use a container(defined in global space) for call `define`/`require` functions in the AMD module.

Reviewer: ```suggestion
Use a container(defined in global space) for calling `define`/`require` functions in an AMD module.
```

> W> Note that value of `amdContainer` **must be** set as global variable.

Reviewer: ```suggestion
W> Note that the value of `amdContainer` **must be** set as a global variable.
```

};
```

> Which would result bundle like below:

Reviewer: ```suggestion
Which will result in the following bundle:
```
